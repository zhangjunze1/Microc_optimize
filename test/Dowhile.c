void main() {
  int i=5;
  do{
      print("%d",i);
      i=i+1;
  } while (i<4);
}
