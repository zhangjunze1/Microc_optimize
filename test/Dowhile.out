24 19 0 5 25 13 0 5 12 13 11 22 15 -1 13 13 11 0 1 1 12 15 -1 13 11 0 4 7 18 9 21 0